presentation/common/src/presentation-common/content/PropertyValueFormatter.ts

```diff
@@ -73,11 +73,11 @@ export class ContentFormatter {
   }
 
   private async formatPropertyValue(value: Value, field: PropertiesField): Promise<DisplayValue> {
-    if (field.isArrayPropertiesField()) {
+    if (field.isArrayPropertiesField() && value !== undefined) {
```

Comment:
I think `value !== undefined` check can be moved to `ContentFormatter.formatValues` function to make sure that we do not try formatting value if it is `undefined`.

Reply:
Moved